native/monogame/desktopvk.vcxproj.filters

```diff
@@ -0,0 +1,86 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Same... this was not intended to be committed as it is generated by premake. 

Let's see what @harry-cpp thinks.

Reply:
I agree, I do not think this file should have been added, please revert